### ISSUE TITLE
Duplicate import statements when same module referenced by mappers and enumValues

### ISSUE THREAD

**Issue by chr1sjf0x** [dotansimha/graphql-code-generator#3257]
**Describe the bug**
The generated code contains duplicate import statements:

```typescript
import { ProjectRole } from '../entities';
import { ... } from 'graphql';
import { ..., ProjectRole, ... } from '../entities';
```

**To Reproduce**

1. Specify a `mappers` entry to any module
2. Specify a `enumValues` entry to the same module as step 1

1. My GraphQL schema:

```graphql
schema {
  query: Query
}

type Query {
  role: [ProjectRoleDetail!]!
}

enum ProjectRole {
  PROJECT_MANAGER
  ETC
}

type ProjectRoleDetail {
  code: ProjectRole!
  name: String!
}
```

2. My GraphQL operations:

```graphql
NA
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: "./schema/**/*.graphql"
documents: null
generates:
  src/generated/graphql-types.ts:
    config:
      contextType: "@src/context#Context"
      useIndexSignature: true
      noSchemaStitching: true
      avoidOptionals: true
      mappers:
        ProjectRoleDetail: ../entities#ProjectRole
      enumValues:
        ProjectRole: ../entities#ProjectRole
    plugins:
      - typescript
      - typescript-resolvers
```

**Expected behavior**
The same module to be imported only once:

```typescript
import { ... } from 'graphql';
import { ..., ProjectRole, ... } from '../entities';
```

**Environment:**

- OS: 
- `@graphql-codegen/1.11.2`: 
- NodeJS: 12.7.0

**Additional context**

We have to items of interest in our schema
1) An enum - ProjectRole
2) A type - ProjectRoleDetail - which acts as a wrapper around the enum and provides additional context, in this case a human readable `name` field

In our database we have a table which represents the enum and name. We are using that data to generate (**ourselves**) the graphql schema as well as a typescript enum and some other helpers.

Next we are trying to configure:

- mappers: From ProjectRoleDetail to **our** generated typescript enum `ProjectRole`
- enumValues: To **NOT** generate it's own `ProjectRole` enum, but instead use **our** generated typescript enum `ProjectRole`

These two configuration options result in duplicate imports

**Comment by dotansimha** [comment]
Thanks @chr1sjf0x , I'm looking into this

**Comment by dotansimha** [comment]
@chr1sjf0x I fixed it in: https://github.com/dotansimha/graphql-code-generator/pull/3385 

Can you please try the latest alpha? `1.11.3-alpha-b3fa829d.91`

**Comment by chr1sjf0x** [comment]
Thank you @dotansimha. It works as expected with `1.11.3-alpha-b3fa829d.91` 👍

**Comment by dotansimha** [comment]
Fixed in `v1.12.0`
